### ISSUE TITLE
Quokka fails to start on files of a second project if multiple project are opened in PhpStorm

### ISSUE THREAD

**Issue by lehtunk** [wallabyjs/quokka#114]
### Quokka.js Console Output

Quokka #1 (node: v8.9.1)

    at Timer.listOnTimeout (timers.js:270:5)

### Code editor version
PhpStorm 2017.2.4
Build #PS-172.4155.41, built on September 15, 2017
Licensed to Hexagon Ragasco AS / Knut Helstad
Subscription is active until December 12, 2017
JRE: 1.8.0_152-release-915-b11 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
### OS name and version

Windows
OSX
Linux

**Comment by ArtemGovorov** [comment]
Is the error displayed for a new/empty file or there's some code that you're running?
If it's a new/empty file, what happens if you add some code to it?

**Comment by lehtunk** [comment]
1. In both cases, the same error message is given.
2. Looks like Quokka is running but gives the same error message.
3. Windows 10 10.0

When I have applied for an error message, it is mentioned in other contexts that Timer return an object instead of a string.

**Comment by lehtunk** [comment]
jsj
Assertion failed: Already disposed
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:168)
	at com.intellij.util.Alarm.checkDisposed(Alarm.java:84)
	at com.intellij.util.Alarm._addRequest(Alarm.java:211)
	at com.intellij.util.Alarm.addRequest(Alarm.java:186)
	at com.quokka.javascript.session.E.a(Unknown Source)
	at com.quokka.javascript.session.E.consume(Unknown Source)
	at com.quokka.javascript.session.QuokkaSession$c.a(Unknown Source)
	at com.quokka.javascript.session.QuokkaSession$c.a(Unknown Source)
	at com.quokka.javascript.session.I.run(Unknown Source)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:288)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:285)
	at com.intellij.util.concurrency.QueueProcessor.lambda$null$0(QueueProcessor.java:93)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:246)
	at com.intellij.util.concurrency.QueueProcessor.lambda$wrappingProcessor$1(QueueProcessor.java:93)
	at com.intellij.util.concurrency.QueueProcessor.lambda$null$3(QueueProcessor.java:226)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:246)
	at com.intellij.util.concurrency.QueueProcessor.lambda$startProcessing$4(QueueProcessor.java:226)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:342)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)

**Comment by ArtemGovorov** [comment]
I tried reproducing the issue in PhpStorm 2017.2.4 on Windows with node 8.9.1, but everything seems to work as expected for me.

Regarding the originally reported error:

```
Quokka #1 (node: v8.9.1)

    at Timer.listOnTimeout (timers.js:270:5)
```

it looks like you have only share a part of the error (look like the error message and maybe a part of the stack is missing). Could you please share the full error, ideally a screenshot of the issue with the opened Quokka file and the output?

**Comment by ArtemGovorov** [comment]
Could you please also try installing [VS Code version](https://marketplace.visualstudio.com/items?itemName=WallabyJs.quokka-vscode) or [Atom version](https://atom.io/packages/atom-quokka) to see if you're getting the same error there? And if so, share the output as well?

**Comment by lehtunk** [comment]
Hello. Found the reason why Quokka does not work with me.

Once I have opened node.js project, I have chosen "Open in current window" and "Add to currently opening projects". Then the right click does not work to "Start Quokka" on an existing file.

But open new scratch file. Quakka works.

When I open node.js project in "new window" Quakka works as it should.

**Comment by ArtemGovorov** [comment]
@lehtunk I still can't reproduce the issue. Here's what I'm doing:
- open any project in the IDE, 
- check that Quokka is working ("Start Quokka" on an existing file and via new scratch file).
- open another project via File menu and select to open it in "This window".
- check that Quokka is working ("Start Quokka" on an existing file and via new scratch file).

What am I missing?

**Comment by lehtunk** [comment]
Not working!
![image](https://user-images.githubusercontent.com/2168056/33018010-ea014596-cdf4-11e7-922e-899ec6415f9f.png)

Working
![image](https://user-images.githubusercontent.com/2168056/33018277-07abe2bc-cdf6-11e7-954d-6bb3cc7659a5.png)

**Comment by ArtemGovorov** [comment]
Thanks for providing the details. The issue is fixed and the fix is published in v1.0.54.
